idea/testData/quickfix/replaceInitializerWithGetter/simple.kt

```diff
@@ -0,0 +1,7 @@
+// "Replace initializer with getter" "true"
+
+fun String.foo() = "bar"
+
+interface A {
+    val name = <caret>"The quick brown fox jumps over the lazy dog".foo()
```

Comment:
Neither the existing intention action nor your suggested quickfix handle `var` properties correctly. They should create a stub for the setter with no implementation.